source/OTRServer/public/display_socket.js

```diff
@@ -0,0 +1,82 @@
+import display_render from "./display_render.js";
+let socket_path = "/game";
+
+// ====================================================================================
+// State Control
+// ====================================================================================
+
+// State logic enum equivalent.
+const State = Object.freeze({
```

Comment:
Can we encapsulate this somewhere and import state in? Reduce the duplicate code etc

Reply:
Yeah I can try that.

Given that the display and user screens also need the same state reference i'll try to move it into the public source.
Will be a bit stinky but should work.